ludwig/combiners/combiners.py

```diff
@@ -122,7 +122,7 @@ def output_shape(self) -> torch.Size:
         return output_tensor["combiner_output"].size()[1:]
 
 
-class BaseCombinerConfig(schema.BaseMarshmallowConfig):
+class BaseCombinerConfig(utils.BaseMarshmallowConfig):
```

Comment:
nit: Consider renaming the file to `schema_utils.py` because `utils` is collision-prone:

https://google.github.io/styleguide/cppguide.html#Namespace_Names

Reply:
Rename the file itself or just alias to to schema_utils?

Reply:
Both, I think. It seems like an improvement if we can avoid having multiple files with the same name (`utils.py`) in the ludwig repo.

Reply:
Sounds good to me. CC @tgaddair @w4nderlust ?

Reply:
Since it's correctly scoped to the `schema` module, I would leave the filename as `utils.py`, but change the import to:

```
from ludwig.schema import utils as schema_utils
```

Does that work?

Reply:
Alternatively, you can import the relevant classes and functions from `utils.py` into `ludwig/schema/__init__.py` and then do:

```
from ludwig import schema
```

I personally like this route a bit more.